pkg/storj/object.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storj
+
+import "time"
+
+// Bucket contains information about a specific bucket
+type Bucket struct {
+	Name    string
+	Created time.Time
+}
+
+// Object contains information about a specific object
+type Object struct {
+	Version  uint32
+	Bucket   string
+	Path     Path
+	IsPrefix bool
+
+	Metadata []byte
+	Created  time.Time
+	Modified time.Time
+
```

Comment:
Where is the expiration time?

Should we make the content type a first class metadata too?

Reply:
We want first-class things that are often used. So `Content-Type` and `Expiration`, both probably yeah.